Jenkinsfile_promote_salt_bundle_packages

```diff
@@ -197,6 +197,67 @@ pipeline {
+
+                echo "Promote project config for Fedora33 subproject"
+                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:Fedora33 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:Fedora33"
+
+                echo "Promote project config for Fedora34 subproject"
+                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:Fedora34 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:Fedora34"
+
+                echo "Promote project config for Fedora35 subproject"
+                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:Fedora35 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:Fedora35"
+
+                echo "Promote project config for Raspbian9 subproject"
+                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:Raspbian9 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:Raspbian9"
+
+                echo "Promote project config for Raspbian10 subproject"
+                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:Raspbian10 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:Raspbian10"
+
+                echo "Promote project config for Raspbian11 subproject"
+                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:Raspbian11 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:Raspbian11"
+
+                echo "Promote project config for openSUSE Leap 15 subproject"
+                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:openSUSE_Leap_15 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:openSUSE_Leap_15"
+
+                echo "Promote project config for openSUSE Tumbleweed subproject"
+                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:openSUSE_Tumbleweed | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:openSUSE_Tumbleweed"
+
+                echo "Promote project config for SLE12 subproject"
+                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:SLE12 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:SLE12"
+
+                echo "Promote project config for SLE15 subproject"
+                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:SLE15 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:SLE15"
+
+                echo "Promote project config for Ubuntu1804 subproject"
+                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:Ubuntu1804 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:Ubuntu1804"
+
+                echo "Promote project config for Ubuntu2004 subproject"
+                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:Ubuntu2004 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:Ubuntu2004"
```

Comment:
```suggestion
                echo "Promote project config for Ubuntu2004 subproject"
                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:Ubuntu2004 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:Ubuntu2004"

                echo "Promote project config for Ubuntu2204 subproject"
                sh "osc meta prjconf systemsmanagement:saltstack:bundle:testing:Ubuntu2204 | osc meta prjconf --file='-' systemsmanagement:saltstack:bundle:Ubuntu2204"
```

Reply:
Oh, I just added it at the same time you put the suggestion :smile: . Thx.